yew/src/agent/worker/private.rs

```diff
@@ -68,7 +90,9 @@ where
         let bridge = PrivateBridge {
             worker,
             _agent: PhantomData,
+            id,
         };
+        send_to_remote::<AGN>(&bridge.worker, ToWorker::Connected(SINGLETON_ID));
```

Comment:
This assumes that worker is loaded already and will receive this message. Imagine this:
1. Initialize worker
2. Send `ToWorker::Connected` to the worker
3. Worker loads and starts listening for messages

Step 2 tried to send a message before the worker was ready to receive it. This is why we need the message queue that you added 😄  We need to wait for the worker to tell us that it is finished loading. After that, we can empty the message queue. The connected message should be the very first message in the queue